service/servicetest/configprovider.go

```diff
@@ -15,27 +15,21 @@
 package servicetest // import "go.opentelemetry.io/collector/service/servicetest"
 
 import (
-	"context"
-
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config"
-	"go.opentelemetry.io/collector/config/configmapprovider"
+	"go.opentelemetry.io/collector/config/configtest"
 	"go.opentelemetry.io/collector/config/configunmarshaler"
 )
 
 // LoadConfig loads a config.Config  from file, and does NOT validate the configuration.
 func LoadConfig(fileName string, factories component.Factories) (*config.Config, error) {
 	// Read yaml config from file
-	cp, err := configmapprovider.NewFile(fileName).Retrieve(context.Background(), nil)
+	cfgMap, err := configtest.LoadConfigMap(fileName)
```

Comment:
This seems to just shuffle things around, with this function still composed entirely of calls to functions from the `config` packages.  What is gained here?  Are further changes planned?

Reply:
> What is gained here?

Less code, and less "untested" code.

> Are further changes planned?

Not in this file or related to the deprecation removal.